src/IAM/UI/REST/Request/LogoutRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\IAM\UI\REST\Request;
+
+use Twitter\IAM\Domain\Auth\Exception\ValidationErrorException;
+
+final readonly class LogoutRequest
+{
+    public function __construct(
+        private string $refreshToken,
+    ) {
+        if ('' === trim($this->refreshToken)) {
```

Comment:
@SergeyGrinenko, why didn't you want to use [beberlei/assert](https://github.com/beberlei/assert?tab=readme-ov-file#assertthat-chaining)?

This check can be that simple:

```suggestion
        Assert::that($value)->notBlank()
```